integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/kotlin/CoroutinesGradleIntegrationTest.kt

```diff
@@ -0,0 +1,48 @@
+package org.jetbrains.dokka.it.gradle.kotlin
+
+import org.gradle.testkit.runner.TaskOutcome
+import org.jetbrains.dokka.it.gradle.AbstractGradleGitIntegrationTest
+import org.jetbrains.dokka.it.gradle.BuildVersions
+import org.jetbrains.dokka.it.gradle.S3Project
+import org.junit.runners.Parameterized
+import java.io.File
+import kotlin.test.*
+
+class CoroutinesGradleIntegrationTest(override val versions: BuildVersions) : AbstractGradleGitIntegrationTest(), S3Project {
+
+    companion object {
+        @get:JvmStatic
+        @get:Parameterized.Parameters(name = "{0}")
+        val versions = BuildVersions.permutations(
+            gradleVersions = listOf("6.3"),
+            kotlinVersions = listOf("1.4.10")
+        )
+    }
+
+    override val projectOutputLocation: File by lazy { File(projectDir, "build/dokka/htmlMultiModule") }
+
+    @BeforeTest
+    fun prepareProjectFiles() {
+        val templateProjectDir = File("projects", "coroutines/kotlinx-coroutines")
+        templateProjectDir.listFiles().orEmpty()
+            .forEach { topLevelFile -> topLevelFile.copyRecursively(File(projectDir, topLevelFile.name)) }
+        copyAndApplyGitDiff(File("projects", "coroutines/coroutines.diff"))
+    }
+
+    @Test
+    fun execute() {
+        val result = createGradleRunner(":dokkaHtmlMultiModule", "-i", "-s").buildRelaxed()
+
+        assertEquals(TaskOutcome.SUCCESS, assertNotNull(result.task(":dokkaHtmlMultiModule")).outcome)
+
+        assertTrue(projectOutputLocation.isDirectory, "Missing dokka output directory")
+
+        projectOutputLocation.allHtmlFiles().forEach { file ->
+//            assertContainsNoErrorClass(file)
+//            assertNoUnresolvedLinks(file)
```

Comment:
is it possible to uncomment those?

Reply:
Nope because it breaks the tests. We should investigate those bugs and fix them in separate PRs